Animations/Zoom/RightBig.js

```diff
@@ -26,5 +28,5 @@ var ZoomRightBig = {
     'transform': 'scale(.1) translateX(2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('ZoomRightBig', ZoomRightBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('ZoomRightBig', ZoomRightBig);
```

Comment:
This does not need to have `'ZoomRightBig'` passed into it since we have defined a `className` property in `ZoomRightBig`.